Sources/Swift/Integrations/UserFeedback/SentryUserFeedbackFormController.swift

```diff
@@ -0,0 +1,204 @@
+//swiftlint:disable type_body_length
+
+import Foundation
+#if os(iOS) && !SENTRY_NO_UIKIT
+@_implementationOnly import _SentryPrivate
+import PhotosUI
+import UIKit
+
+@available(iOS 13.0, *)
+protocol SentryUserFeedbackFormDelegate: NSObjectProtocol {
+    func finished(with feedback: SentryFeedback?)
+}
+
+@available(iOS 13.0, *)
+@objcMembers
+class SentryUserFeedbackFormController: UIViewController {
+    let config: SentryUserFeedbackConfiguration
+    weak var delegate: (any SentryUserFeedbackFormDelegate)?
+    lazy var viewModel = SentryUserFeedbackFormViewModel(config: config, controller: self)
+    
+    override func traitCollectionDidChange(_ previousTraitCollection: UITraitCollection?) {
+        config.theme.updateDefaultFonts()
+        config.recalculateScaleFactors()
+        viewModel.updateLayout()
+    }
+    
+    init(config: SentryUserFeedbackConfiguration, delegate: any SentryUserFeedbackFormDelegate) {
+        self.config = config
+        self.delegate = delegate
+        super.init(nibName: nil, bundle: nil)
+        view.backgroundColor = config.theme.background
+        initLayout()
+        viewModel.themeElements()
+        
+        let nc = NotificationCenter.default
+        nc.addObserver(self, selector: #selector(showedKeyboard(note:)), name: UIResponder.keyboardDidShowNotification, object: nil)
+        nc.addObserver(self, selector: #selector(hidKeyboard), name: UIResponder.keyboardWillHideNotification, object: nil)
+    }
+    
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+}
+
+// MARK: Layout
+@available(iOS 13.0, *)
+extension SentryUserFeedbackFormController {
+    func initLayout() {
+        viewModel.setScrollViewBottomInset(0)
+        view.addSubview(viewModel.scrollView)
+        NSLayoutConstraint.activate(viewModel.allConstraints(view: view))
+    }
+    
+    func showedKeyboard(note: Notification) {
+        guard let keyboardValue = note.userInfo?[UIResponder.keyboardFrameEndUserInfoKey] as? NSValue else {
+            SentryLog.warning("Received a keyboard display notification with no frame information.")
+            return
+        }
+        let keyboardViewEndFrame = self.view.convert(keyboardValue.cgRectValue, from: self.view.window)
+        viewModel.setScrollViewBottomInset(keyboardViewEndFrame.height - self.view.safeAreaInsets.bottom)
+    }
+    
+    func hidKeyboard() {
+        viewModel.setScrollViewBottomInset(0)
+    }
+}
+
+// MARK: SentryUserFeedbackFormViewModelDelegate
+@available(iOS 13.0, *)
+extension SentryUserFeedbackFormController: SentryUserFeedbackFormViewModelDelegate {
+    public func addScreenshotTapped() {
+        
+#if SENTRY_TEST || SENTRY_TEST_CI || TEST || TESTCI
```

Comment:
m: We should not use TEST and TESTCI anymore
```suggestion
#if SENTRY_TEST || SENTRY_TEST_CI
```

Reply:
Thanks, I missed this change in the SDK